pkg/universe.dagger.io/x/solomon@dagger.io/spectral/lint.cue

```diff
@@ -105,10 +111,8 @@ import (
 	container: docker.#Run & {
 		// Default container image can be overrided.
 		//   'spectral' must be installed and in the PATH.
-		input:              docker.#Image | *_buildDefaultImage.output
-		_buildDefaultImage: docker.#Pull & {
-			source: "stoplight/spectral"
-		}
+		input: *#Image | docker.#Image
```

Comment:
I missed a detail in my review, my bad!
See : https://github.com/dagger/dagger/pull/2486#discussion_r884326402